testing/endpoint_v2.go

```diff
@@ -20,10 +22,20 @@ func (endpoint *Endpoint) MsgSendPacket(timeoutTimestamp uint64, packetData chan
 		return channeltypesv2.Packet{}, err
 	}
 
-	// TODO: parse the packet from events instead of manually constructing it. https://github.com/cosmos/ibc-go/issues/7459
-	nextSequenceSend, ok := endpoint.Chain.GetSimApp().IBCKeeper.ChannelKeeperV2.GetNextSequenceSend(endpoint.Chain.GetContext(), endpoint.ChannelID)
-	require.True(endpoint.Chain.TB, ok)
-	packet := channeltypesv2.NewPacket(nextSequenceSend-1, endpoint.ChannelID, endpoint.Counterparty.ChannelID, timeoutTimestamp, packetData)
+	// TODO: parse the packet from events instead of from the response. https://github.com/cosmos/ibc-go/issues/7459
+	// get sequence from msg response
+	var msgData sdk.TxMsgData
```

Comment:
I decided to just port this over, and you can use events later. Subtracting the next seq send felt worse. And this should be an easier template to work off of

Reply:
SGTM :+1:

Reply:
I think we have this helper that might help here

https://github.com/cosmos/ibc-go/blob/5db55dd116d0248b33412fd9985319a45f3849c5/testing/utils.go#L67-L85

but either way will be changed to event parsing later anyway so no big deal

Reply:
True, but i will leave as-is given its changing anyway